packages/charts/src/vaadin-chart.js

```diff
@@ -1081,6 +1081,12 @@ class Chart extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))) {
     if (this.configuration) {
```

Comment:
Could we wrap the `disconnectedCallback` logic inside an async function such as:
```js
requestAnimationFrame(() => {
      if (this.isConnected) {
        return;
      }
```
to avoid the animations from triggering on each move:

https://github.com/user-attachments/assets/4af075a5-f30e-4cda-bc1d-a7d5437abba2

So it would work more like this:

https://github.com/user-attachments/assets/1c4db62b-7d8f-49a5-9b70-aac1fe07a9d3

Reply:
Extracted to #8117, will rebase this PR on top of that one.